usr/sbin/pkcsconf/pkcsconf.c

```diff
@@ -560,6 +561,22 @@ CK_RV check_user_and_group(void)
     return CKR_FUNCTION_FAILED;
 }
 
+void print_info_uri(CK_INFO_PTR info)
+{
+    struct p11_uri *uri;
+
+    uri = p11_uri_new();
+    if (!uri)
+        return;
+
+    uri->info = info;
+
+    printf("\tURI: %s\n", p11_uri_format(uri));
+
+    p11_uri_free(uri);
+    uri = NULL;
```

Comment:
not really required here, but OK (I sometimes also use this to make sure it can't be used afterwards).

Reply:
I will keep it as is.